monitoring/influxdb.go

```diff
@@ -92,11 +195,13 @@ func NewInfluxdbMonitor(config InfluxMonitorConfig, logger log.Logger) (Monitor,
 		}
 	}()
 
-	logger.Info().Log(
-		"msg", fmt.Sprintf("influxdb instrumentation writing to %s://%s@%s/%s", u.Scheme, username, u.Host, monitor.database),
+	go monitor.batchWriteTicker()
```

Comment:
What happens if `monitor.batchWriteTicker` panics?

Reply:
Added `recover` in https://github.com/theplant/appkit/pull/25/commits/6edac7082280b7b75f55382bad87371d22b5a7e4.